programs/cleos/main.cpp

```diff
@@ -538,6 +539,34 @@ void print_action_tree( const fc::variant& action ) {
    }
 }
 
+int get_return_code( const fc::variant& result ) {
+   // if a trx with a processed, then check to see if it failed execution for return value
+   int r = 0;
+   if (result.is_object() && result.get_object().contains("processed")) {
+      const auto& processed = result["processed"];
+      if( processed.get_object().contains( "except" ) ) {
```

Comment:
Do you need to check if `processed` is an object before `get_object()`ing it? You do the `is_object()` check for `result` & `except`. And a failure here with `processed` not being an object would be outside of the try block internal to this function.

Reply:
Yes, that would be better. I copy/pasted from `print_result` which does not have that. I'll add the check.